### PR TITLE
fixes for comments from code reviews

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7515/app_12500_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7515/HighFidelity-Beta-PR12500-87a75e9b70569fb442f55dd88abef0cea878d1d0.dmg

**Comment by birarda** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7518/app_12500_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7519/app_12500_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7519/HighFidelity-Beta-PR12500-d20b775c7a211541305e1d2152ce248bc30b69e5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7519/HighFidelity-Beta-PR12500-d20b775c7a211541305e1d2152ce248bc30b69e5.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7520/app_12500_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7520/HighFidelity-Beta-PR12500-4216ebdd253ec0a37102c1962c9eee26fd92090d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7520/HighFidelity-Beta-PR12500-4216ebdd253ec0a37102c1962c9eee26fd92090d.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7522/app_12500_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7525/app_12500_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7525/HighFidelity-Beta-PR12500-2872412f9c3feb50ae1f630c49f95e38d76ccc2c.dmg

File: domain-server/src/ContentSettingsBackupHandler.cpp [hunk 1]
```diff
@@ -23,20 +23,31 @@ static const QString CONTENT_SETTINGS_BACKUP_FILENAME = "content-settings.json";
...
+    // make a QJsonDocument using the object
     QJsonDocument contentSettingsDocument { contentSettingsJSON };
 
     QuaZipFile zipFile { &zip };
 
-    zipFile.open(QIODevice::WriteOnly, QuaZipNewInfo(CONTENT_SETTINGS_BACKUP_FILENAME));
-    zipFile.write(contentSettingsDocument.toJson());
-    zipFile.close();
+    if (zipFile.open(QIODevice::WriteOnly, QuaZipNewInfo(CONTENT_SETTINGS_BACKUP_FILENAME))) {
+        if (zipFile.write(contentSettingsDocument.toJson()) == -1) {
```
Comment (huffman) [inline domain-server/src/ContentSettingsBackupHandler.cpp]: I think we should circle back and do better handling of this sort of thing. I did something similar on my branch but it potentially leaves the backup with an empty file.

Comment (birarda) [reply domain-server/src/ContentSettingsBackupHandler.cpp]: Agreed - thought at least logging the error was better than not at this point

**Comment by birarda** [comment]
This failed because of that damn PDB API call thing - should be okay to merge into the branch and succeed there.

**Pull request merged**